extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -0,0 +1,94 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+
+@import '../components/_variables';
+
+$background-color-dark: #1a1a1a;
+$background-color-light: #f6f6f6;
+
+$background-color: $background-color-light;
+@if $is-dark-wiki {
+	$background-color: $background-color-dark;
+}
+
+$logo-url-dark: '/extensions/wikia/CommunityPage/images/wikia-logo-dark.svg';
+$logo-url-light: '/extensions/wikia/CommunityPage/images/wikia-logo.svg';
+
+$logo-url: $logo-url-light;
+@if $is-dark-wiki {
+	$logo-url: $logo-url-dark;
+}
+
+.WikiaRail {
+	padding-top: 20px;
+
+	.module {
+		position: relative;
+	}
+}
+
+.community-page-entry-point-module {
+	background: $background-color;
+	position: relative;
+
+	.entry-point-logo {
+		background-image: url($logo-url); /* inline */
+		color: red;
+		height: 30px;
+		left: 50%;
+		position: absolute;
+		top: 0;
+		transform: translate(-50%, -50%);
+		width: 30px;
+	}
+
+	.entry-point-container {
+		@include align-items(center);
+		@include flexbox();
+		@include flex-direction(column);
+
+		font-size: 14px;
+		font-family: $community-page-font-family;
+		padding-top: 5px;
+	}
+
+	.entry-point-button {
+		background: $color-buttons;
+		color: $color-button-text;
+		display: inline-block;
+		font-size: 10px;
+		font-weight: bold;
+		height: 31px;
+		line-height: 31px;
+		padding: 0 30px;
+		text-decoration: none;
+		text-transform: uppercase;
+		white-space: normal;
+
+		&:hover, &:active {
+			background-color: $color-button-highlight;
+			text-decoration: none;
+		}
+	}
+
+	.entry-point-new {
+		background: $color-buttons;
+		border-radius: 8px;
+		color: $color-button-text;
+		font-size: 10px;
+		height: 18px;
+		letter-spacing: 0;
+		line-height: 18px;
+		padding: 0 10px;
+		text-align: center;
+		text-transform: uppercase;
+	}
+
+	.entry-point-grow {
```

Comment:
Name of this selector reflects copy used for message. When copy changes it gets unclear. I think it's better to think about it's UI function.
e.g. .entry-point-description